--- FuzzyOcr-3.5.1/FuzzyOcr.cf~	2007-01-07 14:07:00.000000000 +0200
+++ FuzzyOcr-3.5.1/FuzzyOcr.cf	2007-01-09 16:57:31.029022412 +0200
@@ -1,7 +1,7 @@
 # Syntax:
 # loadplugin <Plugin_Name> <Location>
 #  <Location> path where Plugin resides.
-loadplugin FuzzyOcr FuzzyOcr.pm
+loadplugin FuzzyOcr
 
 body     FUZZY_OCR                   eval:fuzzyocr_check()
 body     FUZZY_OCR_WRONG_CTYPE       eval:dummy_check()
